lib/extensions/ar_adapter/ar_checkin_checkout_callbacks.rb

```diff
@@ -20,3 +20,73 @@ def log_after_checkout
     logger.debug { "#{self.class.name.demodulize}##{__method__}, #{connection_info_for_logging}" } if logger && ActiveRecord::Base.connected?
   end
 end
+
+
+
```

Comment:
I'm sure to easily see it, but [I agree with @Fryguy](https://github.com/ManageIQ/manageiq/pull/12595#issuecomment-260074387), let's throw a conditional to scream when Rails gets updated to remove this stuff.